macos/Runner.xcodeproj/xcshareddata/xcschemes/Runner.xcscheme

```diff
@@ -15,7 +15,7 @@
             <BuildableReference
                BuildableIdentifier = "primary"
                BlueprintIdentifier = "33CC10EC2044A3C60003C045"
-               BuildableName = "Flutter Gallery.app"
+               BuildableName = "gallery.app"
```

Comment:
When built, the app is named `Flutter Gallery.app`, per `macos/Flutter/ephemeral/.app_filename`. Should this also be?

Reply:
Do you know where the `Flutter Gallery` -> `gallery` change came from?  When I `flutter build macos` it updated the project file in the same way, but doesn't touch the buildable name, which is set https://github.com/flutter/gallery/blob/c5e0047f73376d4a2ee9149d9a698e825b0a2ff6/macos/Runner/Configs/AppInfo.xcconfig#L8

Reply:
I tried to reproduce and it fixed itself, my favorite kind of issue. Changing the PRODUCT_NAME correctly propagated to `Runner.xcheme`.